src/components/UserActions/UserActivity.ts

```diff
@@ -97,9 +108,13 @@ export class UserActivity extends Component {
 
         this.userProfileModel.getActions(this.options.userId).then((actions) => {
             const sortMostRecentFirst = (a: UserAction, b: UserAction) => b.timestamp.getTime() - a.timestamp.getTime();
-
             const sortedActions = actions.sort(sortMostRecentFirst);
-            this.sessions = this.splitActionsBySessions(sortedActions);
+
+            let filteredActions = sortedActions;
+            if (this.options.customActionsExclude && this.options.customActionsExclude.length > 0) {
+                filteredActions = actions.filter((action) => this.filterActions(action));
```

Comment:
actions.filter((action) => this.filterActions(action)); or
actions.filter(this.filterActions.bind(this)) might be clearer?

Reply:
Why not this?

```suggestion
                filteredActions = actions.filter(this.filterActions);
```